src/app/routes/idx/getInitialData/index.js

```diff
@@ -0,0 +1,28 @@
+import pipe from 'ramda/src/pipe';
+import fetchPageData from '../../utils/fetchPageData';
```

Comment:
I'm unsure about moving these all to `utils` - was there a `sharedDataTransformers` folder that contains data transformation functions that are used across page types?